library/core/src/panic/location.rs

```diff
@@ -39,7 +39,7 @@ use crate::ptr::NonNull;
 /// See [`Location::file`]'s documentation for more discussion.
 #[lang = "panic_location"]
 #[stable(feature = "panic_hooks", since = "1.10.0")]
-#[cfg_attr(feature = "ferrocene_certified", allow(dead_code))]
+#[cfg_attr(feature = "ferrocene_certified", expect(dead_code))]
```

Comment:
just noting that this is a semantic change. i think it's a good one 👍 but not related to expanding the subset that i can tell.